### DISCUSSION TITLE
Modules installation on linux

### DISCUSSION THREAD

**Discussion by pierre-olivier832** [AlexandreRouma/SDRPlusPlus#1098] [General]
hi. is it possible to get more clear information  on how to install plugins modules for sdr++ under linux? 
Im new on these kind of os (know some terminal command) but im not gifted.
tanck you.

**Comment by AlexandreRouma** [comment]
you just drop the `.so` file in `/usr/lib/sdrpp/plugins`. there aren't many plugins currently available anyway.

**Comment by pierre-olivier832** [comment]
Okay, for that its already do, but i dont find it in module manager.

Obtenir Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by mw1coe** [reply]
I have the same issue.

With Ubuntu 22.04 I have No m17_decoder
With Windows 10 I have no audio_sink
